packages/react-native/package.json

```diff
@@ -7,13 +7,15 @@
   "license": "Apache-2.0",
   "scripts": {
     "build": "yarn dist && cp -a src/assets dist",
+    "build:dist": "tsc --project tsconfig.dist.json",
     "clean": "rimraf dist",
-    "dev": "yarn dist --watch",
-    "dist": "tsc --project tsconfig.dist.json",
-    "lint": "tsc --noEmit && eslint src",
+    "dev": "yarn build:dist --watch",
```

Comment:
Assuming we are doing this to avoid building the tests/mock files in watch mode?

Reply:
Close 😅 We have a second step in the build command copying the assets directory to the dist folder, which would prevent the `--watch` flag from being appended correctly if we were to just run `yarn build --watch`